docs/prod-host-setup.md

```diff
@@ -157,6 +157,36 @@ Additional details of Jailer features can be found in the
 
 ## Host Security Configuration
 
+### Constrain CPU overhead caused by kvm-pit kernel threads
+
+The current implementation results in host CPU usage increase on x86 CPUs when
+a guest injects timer interrupts with the help of kvm-pit kernel thread.
+kvm-pit kthread is by default part of the root cgroup.
+
+To mitigate the CPU overhead we recommend two system level configurations.
+
+1.
+    Use an external agent to move the `kvm-pit/<pid of firecracker>` kernel
+    thread in the microVM’s cgroup (e.g., created by the Jailer).
```

Comment:
nit: maybe it should be more specific like: "kernel thread from root cgroup to the microVM's cgroup"